backend/dal/Migrations/v01.07.02/Up/PostUp/Adminstrative_Areas_And_Addresses.sql

```diff
@@ -0,0 +1,191 @@
+-- Update addresses table first to correct locations which contain District of, City of, or The Corporation of the District of
+PRINT 'Updating Addresses table'
+
+UPDATE dbo.[Addresses]
+SET AdministrativeArea = REPLACE(AdministrativeArea, 'District of ', '')
+where AdministrativeArea like 'District of %';
+
+UPDATE dbo.[Addresses]
+SET AdministrativeArea = REPLACE(AdministrativeArea, 'City of ', '')
+where AdministrativeArea like 'City of %';
+
+UPDATE dbo.[Addresses]
+SET AdministrativeArea = REPLACE(AdministrativeArea, 'The Corporation of the District of ', '')
+where AdministrativeArea like 'The Corporation of the District of %';
+
+-- Do the "one off" updates to simplify the name where there is only once instance of each
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Spallumcheen' 
+WHERE AdministrativeArea = 'The Corporation of the Township of Spallumcheen';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Lake Country' 
+WHERE AdministrativeArea = 'District Of Lake Country';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Port Edward' 
+WHERE AdministrativeArea = 'District of Port Edward';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Highlands' 
+WHERE AdministrativeArea = 'District of Highlands';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Belcarra' 
+WHERE AdministrativeArea = 'Village of Belcarra';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Slocan' 
+WHERE AdministrativeArea = 'Village of Slocan';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Anmore' 
+WHERE AdministrativeArea = 'Village of Anmore';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Warfield' 
+WHERE AdministrativeArea = 'Village of Warfield';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Canal Flats' 
+WHERE AdministrativeArea = 'Village of Canal Flats';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Radium Hot Springs' 
+WHERE AdministrativeArea = 'Village of Radium Hot Springs';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'North Cowichan' 
+WHERE AdministrativeArea = 'The Corporation of the District of North Cowichan';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Coldstream' 
+WHERE AdministrativeArea = 'The Corporation of the District of Coldstream';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Lake Cowichan' 
+WHERE AdministrativeArea = 'Town of Lake Cowichan';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Kent' 
+WHERE AdministrativeArea = 'District of Kent';
+
+
+PRINT 'Updating AdministrativeAreas table'
+
+-- Then update AdministrativeAreas table to fix the Abbreviation field in order to get correct join when doing a batch update later on
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'Valemount' 
+WHERE Name = 'Village of Valemount' AND Abbreviation = 'Valemont';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'Fort St. John' 
+WHERE Name = 'City of Fort St John' AND Abbreviation = 'Fort St John';
```

Comment:
This is a "one-off" case because I'm just updating the Abbreviation to "Fort St. John" with a period so that it can match in the update statement later on which matches the Name to the Abbreviation field.